website/docs/plugin/framework/migrating/index.mdx

```diff
@@ -0,0 +1,51 @@
+
+In addition to this migration guide, we recommend referring to the main [Framework documentation](/plugin/framework) as you migrate your provider.
+
+## Requirements
+
+Before you migrate your provider to the Framework, ensure it meets the following requirements:
+
+- Go 1.18+
+- Built on the latest version of SDKv2
+- The provider is for use with Terraform >= 0.12.0
+
+## Muxing
+
+Consider muxing when you need to migrate a provider that contains many resources or data sources. Muxing lets you use two versions of the same provider concurrently, with each serving different resources or data sources. This lets you migrate individual resources or data sources to the Framework one at a time.
+
+Refer to the [Combining and Translating documentation](/plugin/mux) for details about muxing configuration.
+
+## Testing Migration
+
+As you complete the migration, we recommend that you follow Test Driven Development by writing tests to ensure that the migration does not affect provider behavior. Refer to [Testing Migration](/plugin/framework/migrating/testing/testing#testing-migration) for details and an example.
+
+## Migration steps
+
+Take the following steps when you migrate a provider from SDKv2 to the Framework:
+
+- Ensure all [tests](/plugin/framework/migrating/testing#testing-migration) pass.
+- [Serve the provider](/plugin/framework/migrating/providers#serving-the-provider) via the Framework.
+  - Implement [muxing](/plugin/framework/migrating/providers#muxing) if you plan to migrate the provider over several point releases.
+- Update the [provider definition](/plugin/framework/migrating/providers#provider-definition) to use the Framework.
+- Update the [provider schema](/plugin/framework/migrating/providers#provider-schema).
+- Update each of the provider's resources and data sources.
+  - Update related [tests](/plugin/framework/migrating/testing) to use the Framework, and ensure that the tests fail.
+  - Update the [resource](/plugin/framework/migrating/resources#schema) or [data source](/plugin/framework/migrating/data-sources) schema.
+  - Update the resource or data source's [CRUD function](/plugin/framework/migrating/resources/crud) definitions.
+  - Update the resource or data source's [attribute schema](/plugin/framework/migrating/attributes/schema).
+  - Update the resource's [import function](/plugin/framework/migrating/resources/import), if your provider supports importing the resource.
+  - Implement [plan modification](/plugin/framework/migrating/resources/plan-modification) if needed.
+  - Implement [state upgraders](/plugin/framework/migrating/resources/plan-modification) if needed.
```

Comment:
Should we leave a lot of the details here to the resource overview page? Otherwise, this might be hard to ensure it stays up to date.

```suggestion
  - Migrate the [resource](/plugin/framework/migrating/resources) or [data source](/plugin/framework/migrating/data-sources).
```

If this content remains some of the links here will need to be updated including the "attribute schema" one and "state upgraders" one.